From 99bb5efaee696686e6768a0d1b8f420924c0d9c9 Mon Sep 17 00:00:00 2001
From: Ishimoto Shinobu <nagakamira@gmail.com>
Date: Wed, 13 May 2020 17:10:49 +0000
Subject: [PATCH]  vulkan: Drop use of VK_RESULT_BEGIN_RANGE

This was removed in Vulkan 1.2.140.

> Shortly after 2020-04-24, we will be removing the automatically
> generated `VK_*_BEGIN_RANGE`, `VK_*_END_RANGE`, and `VK_*_RANGE_SIZE`
> tokens from the Vulkan headers. These tokens are currently defined for
> some enumerated types, but are explicitly not part of the Vulkan API.
> They existed only to support some Vulkan implementation internals,
> which no longer require them. We will be accepting comments on this
> topic in [#1230], but we strongly suggest any external projects using
> these tokens immediately migrate away from them.

[#1230]: KhronosGroup/Vulkan-Docs#1230

Part-of: <https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/-/merge_requests/1255>

Signed-off-by: Ishimoto Shinobu <nagakamira@gmail.com>
---
 ext/vulkan/vkerror.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/ext/vulkan/vkerror.c b/ext/vulkan/vkerror.c
index 3fec27e..3dd14dd 100644
--- a/ext/vulkan/vkerror.c
+++ b/ext/vulkan/vkerror.c
@@ -63,8 +63,6 @@ _vk_result_to_string (VkResult result)
 
   if (result >= 0)
     return NULL;
-  if (result < VK_RESULT_BEGIN_RANGE)
-    return "Unknown Error";
 
   for (i = 0; i < G_N_ELEMENTS (vk_result_string_map); i++) {
     if (result == vk_result_string_map[i].result)
-- 
2.26.2

